data/json/items/comestibles/seed.json

```diff
@@ -406,6 +406,43 @@
     "vitamins": [ [ "iron", 2 ], [ "vitC", 20 ] ],
     "seed_data": { "plant_name": "beans", "fruit": "raw_beans", "byproducts": [ "withered" ], "grow": "91 days" }
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "raw_edamame",
+    "copy-from": "seed",
+    "name": "raw edamame",
+    "name_plural": "raw edamame",
+    "weight": 160,
+    "symbol": ".",
+    "color": "green",
+    "healthy": -2,
+    "spoils_in": "3 days",
+    "calories": 56,
+    "description": "Light green raw edamame, you should boil or freeze these, or you could eat them while they are still toxic.  Alternatively, you could plant them.",
+    "price": 150,
+    "fun": 1,
+    "vitamins": [ [ "iron", 26 ], [ "vitC", 16 ], [ "calcium", 6 ] ],
+    "seed_data": { "plant_name": "raw_edamame", "fruit": "raw_edamame", "byproducts": [ "withered" ], "grow": "60 days" }
```

Comment:
I looked up what edamames are, and they seem to be the same soybeans, except immature and in the pod. This means that you can't really plant them, and also that they're the same plant as soybeans (so edamames are likely to be a byproduct for growing soybean plant, since harvesting plants at different times can't yield different results yet).